teeio-validator/library/cxl_ide_test_lib/test_case/ide_km_common.c

```diff
@@ -42,39 +42,51 @@ void cxl_dump_key_iv_in_rp(const char* direction, uint8_t *key, int key_size, ui
   TEEIO_DEBUG((TEEIO_DEBUG_INFO, "  %s IF: 0x%016"PRIx64"\n", direction, *(uint64_t *)iv));
 }
 
+bool static gen_default_iv(uint32_t* iv, int iv_size)
+{
+  if(iv_size != 12) {
+    TEEIO_ASSERT(false);
+    return false;
+  }
+
+  iv[0] = CXL_IDE_KM_KEY_SUB_STREAM_CXL<<24;
+  iv[1] = 0;
+  iv[2] = 1;
+
+  return true;
+}
+
 /**
  * Generate CXL-IDE Key/IV.
  */
 static bool cxl_ide_generate_key(const void *pci_doe_context,
                                   void *spdm_context, const uint32_t *session_id,
                                   uint8_t stream_id, uint8_t key_sub_stream, uint8_t port_index,
                                   cxl_ide_km_aes_256_gcm_key_buffer_t *key_buffer,
-                                  bool key_iv_gen_capable, uint8_t direction,
+                                  bool key_gen_capable, bool iv_gen_capable, uint8_t direction,
                                   uint8_t *cxl_ide_km_iv
                                   )
 {
   bool result = true;
   libspdm_return_t status;
+  bool key_or_iv_capable = key_gen_capable || iv_gen_capable;
```

Comment:
this is only used once?

Reply:
Yes.
Will remove this variable ```key_or_iv_capable``` and use ```key_gen_capable || iv_gen_capable``` directly instead.